catalyst/contrib/nn/criterion/__init__.py

```diff
@@ -1,5 +1,6 @@
 # flake8: noqa
 
+import catalyst
```

Comment:
why do we need it here?

Reply:
[delete](https://github.com/catalyst-team/catalyst/pull/1293/commits/e3a0fb7348352a40fd041fd0e3135fe1f2f5d3fd)